user_guide_src/source/changelogs/v4.1.6.rst

```diff
@@ -33,6 +33,7 @@ Enhancements
 ************
 
 - Database pane on debug toolbar now displays location where Query was called from. Also displays full backtrace.
+- :ref:`Subqueries<query-builder-where-subquery>` in QueryBuilder can now be an instance of the BaseBuilder class.
```

Comment:
We insert a space. This seems to work without it, but please add it for consistency.

```suggestion
- :ref:`Subqueries <query-builder-where-subquery>` in QueryBuilder can now be an instance of the BaseBuilder class.
```